azurelinuxagent/common/osutil/openwrt.py

```diff
@@ -126,7 +126,7 @@ def restart_ssh_service(self): # pylint: disable=R1710
         if os.path.exists("/etc/init.d/sshd"): # pylint: disable=R1705
             return shellutil.run("/etc/init.d/sshd restart", chk_err=True)
         else:
-            logger.warn("sshd service does not exists", username) # pylint: disable=E0602
+            logger.warn("sshd service does not exists")
```

Comment:
This seems like this was just copy-pasted from a method with `username` in scope.